.circleci/config.yml

```diff
@@ -268,6 +268,7 @@ workflows:
     jobs:
       - hmpps/npm_security_audit:
           slack_channel: << pipeline.parameters.alerts-slack-channel >>
+          node_tag: << pipeline.parameters.node-version >>
```

Comment:
this ensures npm security audit job runs at the same version as our project's node version